src/tv2-common/actions/executeAction.ts

```diff
@@ -761,7 +765,11 @@ async function executeActionSelectDVELayout<
 		}
 
 		const newMetaData: DVEPieceMetaData = {
-			type: Tv2PieceType.SPLIT_SCREEN,
+			playoutContent: {
```

Comment:
It just looks better this way, i really like that we have this way of describing it now. Nice!